spotbugs/etc/messages_ja.xml

```diff
@@ -7,16 +7,16 @@
   **********************************************************************
   -->
   <Plugin>
-    <ShortDescription>コア FindBugs プラグイン</ShortDescription>
+    <ShortDescription>コア SpotBugs プラグイン</ShortDescription>
     <Details>
 <![CDATA[
 <p>
-このプラグインにはすべての標準的な FindBugs ディテクタがあります。
+このプラグインにはすべての標準的な SpotBugs ディテクタがあります。
 </p>
 ]]>
     </Details>
-    <BugsUrl>https://spotbugs.github.io/bugDescriptions_ja.html</BugsUrl>
-    <AllBugsUrl>https://spotbugs.github.io/bugDescriptions_ja.html</AllBugsUrl>
+    <BugsUrl>https://spotbugs.readthedocs.io/ja/latest/bugDescriptions.html</BugsUrl>
+    <AllBugsUrl>https://spotbugs.readthedocs.io/ja/latest/bugDescriptions.html</AllBugsUrl>

```

Comment:
could you propose the same fix for English version?

Reply:
OK.
Fix hyperlinks with another PR.